### PR TITLE
D8CORE-4871: created a responsive table for vpue

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Creating a responsive table 

# Review By (Date)
- 10/21

# Criticality
- Normal
- VPUE Subtheme

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Check out this PR: https://github.com/SU-SWS/stanford_profile/pull/460
2. Rebuild Cache and import config `drush cr ; drush ci` 
3. Navigate to page with text paragraph. 
4. Add a table and add the `class="resp_table"` to the table
4. Verify it works.
5. Desktop
image
Mobile
image

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? YES. It is in the Stanford_Profile.
- https://github.com/SU-SWS/stanford_profile/pull/460

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Needs : https://github.com/SU-SWS/stanford_profile/pull/460 
- VPUE Subthem

# Associated Issues and/or People
- D8CORE-4871
- https://github.com/SU-SWS/stanford_profile/pull/460
- cc: FYI, @pookmish 

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by jenbreese** [comment]
Hi @jdwjdwjdw, I updated this PR to remove the loss of the table header.  Here is a sample table. 
`<table class="resp_table">
  <caption>This table shows communication items, dates, and actions transfer students should complete between May and September.</caption>
	<thead>
    <tr>
      <th>Communication Item</th>
			<th>Date</th>
			<th>What to Do</th>
		</tr>
  </thead>
  <tbody>
    <tr>
      <td>
				<p><strong>First Approaching Stanford Email</strong></p>
			</td>
			<td>
				<p>Friday, June 5, 2020</p>
			</td>
			<td>
				<p>Check your <a href="http://webmail.stanford.edu">Stanford email</a> for <a href="http://undergrad.stanford.edu/advising/approaching-stanford/approaching-stanford-newsletter">weekly newsletters</a> beginning June 5. You are responsible for reading your email and completing any required actions.</p>
				<p>Log into the Approaching Stanford Portal in <a href="http://canvas.stanford.edu">Stanford Canvas</a> to review the "Approaching Stanford Requirements Checklist" to complete this summer.</p>
				<p>If needed, log in to <a href="http://axess.stanford.edu/">Axess </a>and update your Preferred Name and Mailing Address in the "Student" tab.</p>
				<p>Read the <a href="https://view.publitas.com/stanford-undergrad/approaching-stanford-2020-21">Approaching Stanford Handbook</a>.</p>
			</td>
		</tr>
    <tr>
      <td>
				<p><a href="undergrad.stanford.edu/advising/approaching-stanford/approaching-stanford-forms-instructions">Approaching Stanford Forms Deadline</a></p>
			</td>
			<td>
				<p>Friday, June 26 by noon (PDT)</p>
			</td>
			<td >
				<p>Complete the required <a href="/advising/approaching-stanford/approaching-stanford-forms-instructions">Approaching Stanford Forms</a> online and <a href="https://accessories.stanford.edu/sc/photo/submission">upload your photo</a> for your Stanford ID card. These forms also collect your family contact information for the university. Transfer students living in undergraduate housing apply for housing on these forms.</p>
				<p>Graduate Housing: If you are eligible to live in grad housing, please download the appropriate application and email it to <a href="http://studenthousing@stanford.edu">studenthousing@stanford.edu</a> by July 7.</p>
				<ul><li><a href="https://stanford.box.com/shared/static/7q9s935lzbfsvut5hc0rc5537h42x1h0.pdf">Age 25+ Single Housing Application</a></li>
					<li><a href="https://stanford.box.com/shared/static/qfur0ihm66fk6mplwltrk09jqj36p3g5.pdf">Couple and/or Family Housing Application</a></li>
				</ul></td>
		</tr>
    <tr>
      <td>
				<p><a href="https://admission.stanford.edu/apply/transfer/transcripts.html">Final Official Transcript &amp; Test Scores Deadline</a></p>
			</td>
			<td>
				<p>July 15</p>
			</td>
			<td>
				<p>Instructions on <a href="https://admission.stanford.edu/apply/transfer/transcripts.html">submitting your final college transcript</a>, if applicable, to the Office of Undergraduate Admission are in your admission packet.  <a href="mailto:credentials@stanford.edu">Email credentials@stanford.edu</a> with questions.</p>
				<p>Contact the <a href="https://www.collegeboard.org/">College Board</a> or <a href="https://www.act.org/">ACT</a> and have your official SAT or ACT scores sent to Stanford.</p>
			</td>
		</tr>
    <tr>
      <td>
				<p><a href="/advising/approaching-stanford/approaching-stanford-parents-and-guardians">Parent and Family Email</a></p>
			</td>
			<td>
				<p>Mid-July</p>
			</td>
			<td>
				<p>The first of this monthly newsletter will be emailed to the parent(s) or family member(s) you listed on your <a href="/advising/approaching-stanford/approaching-stanford-forms/approaching-stanford-forms">Approaching Stanford Forms</a>. Review the <a href="https://undergrad.stanford.edu/advising/approaching-stanford/approaching-stanford-newsletter">newsletter archive</a> online.</p>
			</td>
		</tr>
    <tr>
      <td>
				<p><strong>Vice Provost for Undergraduate Education Email</strong></p>
			</td>
			<td>
				<p>August 19</p>
			</td>
			<td>
				<p>VPUE email introduction of Stanford Introductory Seminars and Transfer 101 Course. View the <a href="http://exploreintrosems.stanford.edu/">2020-21 Introductory Seminars online catalog</a> and sign up for Autumn IntroSems by the end of August. Transfers have priority for sophomore-preference seminars. (UPDATED)</p>
			</td>
		</tr>
    <tr>
      <td >
				<p><a href="http://sfs.stanford.edu/student-accounts/pay-your-bill/understanding-your-student-account">University Bill Payment Deadline</a></p>
			</td>
			<td>
				<p>October 15</p>
			</td>
			<td>
				<p>Beginning September 18, view your first <a href="https://sfs.stanford.edu/student-accounts/pay-your-bill/understanding-your-student-account">University bill</a> online in <a href="http://axess.stanford.edu/">Axess</a> and <a href="http://sfs.stanford.edu/student-accounts/pay-your-bill">pay your bill</a>. Set up <a href="http://sfs.stanford.edu/student-accounts/refunds/direct-deposit">Direct Deposit</a> and <a href="https://sfs.stanford.edu/student-accounts/pay-your-bill">ePay</a> in Axess. Your first University Bill is <strong>due October 15</strong>. (UPDATED)</p>
			</td>
		</tr>
  </tbody>
</table>`

**Review by jdwjdwjdw — APPROVED** [review]
G2G

File: src/scss/theme/_tables.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
+@charset "UTF-8";
+
+// Special table for VPUE.
+.resp_table { 
+    width: 100%;
```
Comment (pookmish) [inline src/scss/theme/_tables.scss:5]: Scss should be indented with 2 spaces, not 4.

Comment (jenbreese) [reply src/scss/theme/_tables.scss:5]: @pookmish Oops. I'll fix.

**Pull request merged**